File: app/src/main/java/io/lerk/lrkFM/exceptions/NoAccessException.java
package io.lerk.lrkFM.exceptions;

import java.io.Serializable;

/**
 * {@link NoAccessException}.
 *
 * @author Lukas Fülling (lukas@k40s.net)
 */
public class NoAccessException extends Exception implements Serializable {
    static final long serialVersionUID = 10L;

    public NoAccessException(String message) {
        super(message);
    }
}

File: app/src/main/java/io/lerk/lrkFM/exceptions/NoAccessException.kt
package io.lerk.lrkFM.exceptions

import java.io.Serializable

/**
 * [NoAccessException].
 *
 * @author Lukas Fülling (lukas@k40s.net)
 */
class NoAccessException(message: String?) : Exception(message), Serializable {
    companion object {
        const val serialVersionUID = 10L
    }
}
